rpcs3/Emu/Cell/lv2/sys_memory.cpp

```diff
@@ -268,7 +268,11 @@ error_code sys_memory_container_get_size(vm::ptr<sys_memory_info_t> mem_info, u3
 	// This buffer seems to be used by the PS3 OS for c style 'mallocs'
 	// Todo: Research this more, even though we dont use this buffer, it helps out games when calculating
 	//	     expected memory they can use allowing them to boot
-	mem_info->available_user_memory = ct->size - ct->used - 0x1000000;
+	mem_info->available_user_memory = ct->size - ct->used;
+	if (mem_info->available_user_memory >= 0x1000000)
+	{
+		mem_info->available_user_memory -= 0x1000000;
```

Comment:
That's not what happens, read the comment above this snippet.

Reply:
why shouldn't the Available user memory be capped or at least 'fixed when it's read into available_user_memory var, instead of pushing back the buffer size if it exceeds a certain amount? i.e., why not just set buffer to 0xFFFFFF